gateway/1.4/modules/ROOT/pages/flex-gateway-k8-ingress-class.adoc

```diff
@@ -0,0 +1,61 @@
+= Configure Flex Gateway as an Ingress Controller in Local Mode
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+For Kubernetes deployments in Local Mode, Flex Gateway typically acts as an ingress controller that manages external access to your Kubernetes cluster.
+
+To configure Flex Gateway as an ingress controller, you must configure an `IngressClass` resource and an `Ingress` resource. The `IngressClass` resource configures Flex Gateway as an ingress controller, and the `Ingress` resources configures the services Flex Gateway manages as an ingress controller.
```

Comment:
1. I'm not sure about this first sentence, I feel like it gives out the idea that the customer must do something outside of the install for this to work, ingress class is automatically created.
2. You don't need ingress resources as part of your config, once you have flex installed and good to go, it consumes ingress resources, as it could consume api instances for example.

Maybe something like
`Flex gateway configures itself as an IngressClass by default. It processes Ingress resources that are pointing to flex's IngressClass, acting as an ingress controller.`